profiles/local-frontend.js

```diff
@@ -9,5 +9,6 @@ routes[`/beta/${SECTION}/${APP_ID}`] = { host: `https://localhost:${FRONTEND_POR
 routes[`/${SECTION}/${APP_ID}`]      = { host: `https://localhost:${FRONTEND_PORT}` };
 routes[`/beta/apps/${APP_ID}`]       = { host: `https://localhost:${FRONTEND_PORT}` };
 routes[`/apps/${APP_ID}`]            = { host: `https://localhost:${FRONTEND_PORT}` };
+routes[`/api/ros/v0`]                = { host: `http://localhost:5000` };
```

Comment:
As per my understanding this config is specific to backend & useful while running server locally.

Will it be good to maintain this change under [RedHatInsights/ros-backend](https://github.com/RedHatInsights/ros-backend)?